src/todo/components/add-input/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @flow
+ */
+// $FlowFixMe  (Flow doesn't like importing html files)
+import template from './partial.html';
+
+const component = function(addHandler: Function) {
```

Comment:
not really a fan of this, hopefully handlebars will be able to make this cleaner (with types?)

Reply:
I guess we could set return types on these functions though, and define/export a type in this file that defines the component...

Reply:
Tried this, but it didn’t really work with flow... maybe need to think about typescript